docs/zh-cn/release/download.md

```diff
@@ -9,15 +9,15 @@
 ### DolphinScheduler
 | 日期 | 版本| | 下载 |
 |:---:|:--:|:--:|:--:|
-| Feb. 24th, 2020 | 1.2.1 | Source code| [[src]](https://www.apache.org/dyn/closer.cgi/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-src.zip) [[asc]](https://www.apache.org/dist/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-src.zip.asc) [[sha512]](https://www.apache.org/dist/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-src.zip.sha512)|
-| | | Binary Distribution| [[tar]](https://www.apache.org/dyn/closer.cgi/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-dolphinscheduler-bin.tar.gz) [[asc]](https://www.apache.org/dist/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-dolphinscheduler-bin.tar.gz.asc) [[sha512]](https://www.apache.org/dist/incubator/dolphinscheduler/1.2.1/apache-dolphinscheduler-incubating-1.2.1-dolphinscheduler-bin.tar.gz.sha512)|
-| Jan. 2th, 2020 | 1.2.0 | Source code| [[src]](https://www.apache.org/dyn/closer.cgi/incubator/dolphinscheduler/1.2.0/apache-dolphinscheduler-incubating-1.2.0-src.zip) [[asc]](https://www.apache.org/dyn/closer.cgi/incubator/dolphinscheduler/1.2.0/apache-dolphinscheduler-incubating-1.2.0-src.zip.asc) [[sha512]](https://www.apache.org/dyn/closer.cgi/incubator/dolphinscheduler/1.2.0/apache-dolphinscheduler-incubating-1.2.0-src.zip.sha512)|
```

Comment:
[Here](https://blogs.apache.org/infra/entry/more-secure-and-robust-downloads) is the source from INFRA team. Basically for release artifact, i.e., the tarball, it should be linked to the mirror system; while signature & items for verification, should be linked to downloads.a.o.